src/pyaro/timeseries/Filter.py

```diff
@@ -1010,7 +1010,7 @@ def __init__(
         self._UNITS_METER = None
 
     @property
-    def UNITS_METER(self):
+    def UNITS_METER(self):  #: :meta hide-value:
```

Comment:
These are still shown in the docs: https://pyaro--70.org.readthedocs.build/70/api.html#pyaro.timeseries.Filter.RelativeAltitudeFilter.UNITS_METER

Maybe `:meta private:` will work

Reply:
Thanks